libseccomp/src/notify.rs

```diff
@@ -33,11 +35,32 @@ fn notify_supported() -> Result<()> {
 /// Represents a file descriptor used for the userspace notification.
 pub type ScmpFd = RawFd;
 
+bitflags! {
+    /// Userspace notification response flags
+    pub struct ScmpNotifRespFlags: u32 {
+        /// Tells the kernel to continue executing the system call that triggered the
+        /// notification. Must only be used when the notification response's error is 0.
```

Comment:
Regarding the last sentence, is the following description better?

`Must only be used when the notification response's error and value are 0.`

Ref. https://man7.org/linux/man-pages/man2/seccomp_unotify.2.html
> SECCOMP_USER_NOTIF_FLAG_CONTINUE and the error and val fields must be zero.